src/panels/config/integrations/integration-panels/dynalite/dynalite-config-panel.ts

```diff
@@ -0,0 +1,279 @@
+import "@material/mwc-button";
+import "@polymer/app-layout/app-header/app-header";
+import "@polymer/app-layout/app-toolbar/app-toolbar";
+import "@polymer/paper-input/paper-input";
+import "@polymer/paper-listbox/paper-listbox";
+import "@polymer/paper-item/paper-item";
+import {
+  css,
+  CSSResultArray,
+  customElement,
+  html,
+  LitElement,
+  property,
+  internalProperty,
+  TemplateResult,
+} from "lit-element";
+import "../../../../../components/ha-card";
+import "../../../../../components/ha-menu-button";
+import "../../../../../layouts/ha-app-layout";
+import "../../../../../components/ha-settings-row";
+import "../../../../../components/ha-paper-dropdown-menu";
+import "../../../../../components/ha-switch";
+import { haStyle } from "../../../../../resources/styles";
+import { HomeAssistant } from "../../../../../types";
+import { getConfigEntries } from "../../../../../data/config_entries";
+
+@customElement("dynalite-config-panel")
+class HaPanelDevDynalite extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property() public narrow!: boolean;
+
+  private entryData;
```

Comment:
```suggestion
  private _entryData;
```
Private variables should be prefaced with `_`

Also just from a code style POV move the non property private variables below the properties